src/index.spec.js

```diff
@@ -939,13 +956,10 @@ describe('Dropzone', () => {
 
     it('should not throw error when preview cannot be created', async () => {
```

Comment:
We could rename this test case to `should not generate previews for non-File items`